### PR TITLE
Change dataset_type VARIANTS to SNV_INDEL

### PR DESCRIPTION
**Pull request by hanars**

### CODE REVIEW

File: ui/shared/utils/constants.js [hunk 1]
```diff
@@ -111,7 +111,7 @@ export const MATCHMAKER_CONTACT_URL_FIELD = {
 
 // SAMPLES
 
-export const DATASET_TYPE_VARIANT_CALLS = 'VARIANTS'
+export const DATASET_TYPE_VARIANT_CALLS = 'SNV_INDEL'
```
Comment (bpblanken) [inline ui/shared/utils/constants.js:114]: Is it painful to also rename the constant `DATASET_TYPE_SNV_INDEL_CALLS`?

File: seqr/migrations/0054_alter_sample_dataset_type.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.2.18 on 2023-09-15 14:54
+
+from django.db import migrations, models
+
+OLD_DT = 'SNV_INDEL'
```
Comment (bpblanken) [inline seqr/migrations/0054_alter_sample_dataset_type.py:5]: Are `OLD_DT` and `NEW_DT` meant to be the same?

File: ui/shared/utils/constants.js (hunk 1 above)
Comment (hanars) [reply ui/shared/utils/constants.js:114]: nope! thats a good idea

File: seqr/migrations/0054_alter_sample_dataset_type.py (hunk 2 above)
Comment (hanars) [reply seqr/migrations/0054_alter_sample_dataset_type.py:5]: 🤦 no, there was an overzealous refactor that I failed to fully undo. I tested it locally with them being different, and then later this got changed, I will change it back

**Review by bpblanken — APPROVED** [review]

**Pull request merged**